eng/scripts/Language-Settings.ps1

```diff
@@ -4,15 +4,16 @@ $PackageRepository = "NPM"
 $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
+$ArtifactToPackgeMap = @{'opentelemetry-exporter-azure-monitor'='microsoft-opentelemetry-exporter-azure-monitor'}
```

Comment:
I don't like this approach. We should see if we can figure out another way to get this information. Even if we have to parse it out of the package we should consider that.

Reply:
I prefer to avoid this mapping change. But only files available when processing docs is html documentation file. One alternative option I have thought is to parse global.html file in documentation artifact to get the package title. But this isn't either reliable if documentation format is changed in future. Here in this case what we have is a partial artifact name. So another option is to go through all packages in JS repo and match it against microsoft-<basename> and azure-<basename> to find existing documentation in storage account.

Reply:
See my other comment we are already parsing the package to get the release tag so we should be able to do the same thing to get the package name.